common/v2/routing/routes.tsx

```diff
@@ -1,37 +1,144 @@
-import React from 'react';
+import React, { lazy } from 'react';
 import { Redirect } from 'react-router-dom';
 
-import { IAppRoute } from '@types';
-import { ROUTE_PATHS, IS_ACTIVE_FEATURE } from '@config';
-import {
-  AddAccountFlow,
-  CreateWallet,
-  Mnemonic,
-  Keystore,
-  Dashboard,
-  Settings,
-  Import,
-  Export,
-  DownloadApp,
-  NoAccounts,
-  ScreenLockNew,
-  ScreenLockLocked,
-  ScreenLockForgotPassword,
-  SendAssets,
-  ReceiveAssets,
-  SwapAssetsFlow,
-  Home,
-  SignAndVerifyMessage,
-  BroadcastTransactionFlow,
-  InteractWithContractsFlow,
-  DeployContractsFlow,
-  DeFiZapFlow,
-  PurchaseMembershipStepper,
-  MembershipEducation,
-  BuyAssets
-} from '@features';
+import { IAppRoute } from 'v2/types';
+import { ROUTE_PATHS, IS_ACTIVE_FEATURE } from 'v2/config';
+import { Home, Dashboard, NoAccounts } from 'v2/features';
 import { requiresDesktopApp } from './helpers';
 
+const CreateWallet = lazy(() =>
+  import(
+    /* webpackChunkName: "CreateWallet" */
+    'v2/features/CreateWallet/CreateWallet'
+  )
+);
+const AddAccountFlow = lazy(() =>
+  import(
+    /* webpackChunkName: "AddAccountFlow" */
+    'v2/features/AddAccount/AddAccountFlow'
+  )
+);
+const SendAssets = lazy(() =>
+  import(
+    /* webpackChunkName: "SendAssets" */
+    'v2/features/SendAssets/SendAssets'
+  )
+);
+const Mnemonic = lazy(() =>
+  import(
+    /* webpackChunkName: "Mnemonic" */
+    'v2/features/CreateWallet/Mnemonic/Mnemonic'
+  )
+);
+const Keystore = lazy(() =>
+  import(
+    /* webpackChunkName: "Keystore" */
+    'v2/features/CreateWallet/Keystore/Keystore'
+  )
+);
```

Comment:
FYI: These (and their dependencies are electron only.

Reply:
Ok, they will be lazy loaded anyway, in case the route is hit.